app/controllers/slides_controller.rb

```diff
@@ -165,6 +165,11 @@ def embedded
       @slide.increment(:embedded_view).increment(:total_view).save
     end
     @start_position = slide_position
+    if params.has_key?(:inside) && params[:inside] == '1'
```

Comment:
__[rubocop]__
* Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.